libbeat/docs/keystore.asciidoc

```diff
@@ -35,7 +35,7 @@ For example, imagine that the keystore contains a key called `ES_PWD` with the
 value `yourelasticsearchpassword`:
 
 * In the configuration file, use `output.elasticsearch.password: "${ES_PWD}"`
-* On the command line, use: `-E "output.elasticsearch.password=${ES_PWD}"`
+* On the command line, use: `-E "output.elasticsearch.password=$\{ES_PWD}"`
```

Comment:
@dedemorton I just realize I've made a mistake in the original review, we need to escape the `$` not the `{`:

The following will work `-E "output.elasticsearch.password=\${ES_PWD}"` or use single quote.

Reply:
Yah that was a typo. Thanks for catching this!